### PR TITLE
Add id for slot event to v2 MultiSlotEvent.fbs

### PR DESCRIPTION
**Pull request by olgavrou**

### CODE REVIEW

File: rlclientlib/schema/v2/MultiSlotEvent.fbs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 namespace reinforcement_learning.messages.flatbuff.v2;
 
 table SlotEvent {
+    decision_slot_id:string;
```
Comment (ataymano) [inline rlclientlib/schema/v2/MultiSlotEvent.fbs:5]: [nit] maybe just "id" is better here? In MultiSlotEvent->Slots->DecisionSlotId construction "Slot" seems too repetitive + not 100% about the role of Decision prefix

**Review by ataymano — APPROVED** [review]

File: rlclientlib/schema/v2/MultiSlotEvent.fbs (hunk 1 above)
Comment (olgavrou) [reply rlclientlib/schema/v2/MultiSlotEvent.fbs:5]: Good point about removing `slot`. I am also not sure about `decision_id` vs `id` as to not confuse it with the slates `slot_id` and make it clear that this is coming from the decision service

Comment (kumpera) [reply rlclientlib/schema/v2/MultiSlotEvent.fbs:5]: New fields must *always* got at the end of a table. See https://google.github.io/flatbuffers/flatbuffers_guide_writing_schema.html

Additionally, please document with a comment those fields that are new.

**Review by kumpera — CHANGES_REQUESTED** [review]

File: rlclientlib/schema/v2/MultiSlotEvent.fbs (hunk 1 above)
Comment (olgavrou) [reply rlclientlib/schema/v2/MultiSlotEvent.fbs]: Done

**Pull request merged**